homework4/test/test_task02.py

```diff
@@ -0,0 +1,22 @@
+from unittest.mock import Mock
+from urllib.request import URLError
+
+import pytest
+
+from homework4.task02 import count_dots_on_i
+
+url = "mocked"
+
+
+def test_count_dots_on_i_positive():
+    mocked_count_dots_on_i = Mock(count_dots_on_i, return_value=11)
```

Comment:
u dont need to mock the whole function.. no sense in these tests.. u need to mock a html text which should be return from NET